tools/topology/sof-apl-keyword-detect.m4

```diff
@@ -0,0 +1,91 @@
+#
+# Topology for ApolloLake with direct attach digital microphones array for
+# keyword detection and triggering use case.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Apollolake DSP configuration
+include(`platform/intel/bxt.m4')
+include(`platform/intel/dmic.m4')
+
+DEBUG_START
+
+#
+# Define the pipelines
+#
+# PCM 0 <-------+- KPBM 0 <-- B0 <-- DMIC6 (DMIC01)
+#               |
+# Keyword <-----+
+
+dnl PIPELINE_PCM_ADD(pipeline,
+dnl     pipe id, pcm, max channels, format,
+dnl     frames, deadline, priority, core)
+
+# Passthrough capture pipeline 1 on PCM 0 using max 2 channels.
+# Schedule 320 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-kfbm-capture.m4,
+	1, 0, 2, s32le,
+	320, 1000, 0, 0)
```

Comment:
@keyonjie 20000 for the deadline? This is why we need the macro :)

Reply:
I think the deadline is not the period team, it is for scheduler, so no need to change deadline accordingly?

Reply:
@keyonjie the deadline should change too to maintain 16KHz isnt it?

Reply:
@tlauda can you confirm that if we need change deadline for different period size?

Reply:
@tlauda @keyonjie this PR is ready for merging after confirming if the deadline needs to change or not.

Reply:
@ranj063 @keyonjie Pipeline will be scheduled every 20 ms, so the deadline should be also increased to 20000. 1000 will also work, but in case of many pipelines it's better not to risk it.

Reply:
@tlauda if so, we actually don't need this param passed in to FW, we can set it to period_time inside FW directlly.

Reply:
anyway, let me change to use macros for it ATM.